src/element/resizeElements.ts

```diff
@@ -235,6 +235,26 @@ const measureFontSizeFromWH = (
   return null;
 };
 
+const getSidesForResizeHandle = (
+  resizeHandle: "n" | "s" | "w" | "e" | "nw" | "ne" | "sw" | "se",
+  isResizeFromCenter: boolean,
+) => {
+  return {
+    n:
+      /^(n|ne|nw)$/.test(resizeHandle) ||
```

Comment:
If we really want, we could do this:
```suggestion
      /n/.test(resizeHandle) ||
```

Reply:
yea, or `/^n[ew]?$/` if we wanted to be safe.. dunno